docs/blog/posts/bids-validator-2.0.md

```diff
@@ -0,0 +1,398 @@
+are prevalent and valued in the neuroimaging community, most of that expertise is in Python,
+R and MATLAB (and to some extent C/C++), and relatively few researchers have much experience
+with Javascript. As a result, as BIDS grew with BIDS Extension Proposals (BEPs), updating the
+validator to support the new data types and enforce the new rules became a consistent
+bottleneck.
+
+### The BIDS Schema
+
+In the summer of 2021, the BIDS [Steering Group][] approved the development of the BIDS Schema,
+a project to encode as much of the BIDS Standard as possible in declarative,
+machine-readable rules, and to rewrite the validator to use the schema to enforce the rules.
+This would have several advantages:
+
+1.  A schema could be updated by BIDS contributors without knowledge of Javascript or an
+    understanding of the structure of the BIDS Validator code base.
+
+1.  A language agnostic schema would enable validators to be written in other languages,
+    or for other tools to be written in a way where updates to BIDS could be automatically
+    incorporated by upgrading the schema.
+
+1.  By using the schema as an authoritative source of definitions,
+    large portions of the *text* of the BIDS specification could be generated automatically,
+    and with less chance for inconsistency.
+
+A description of the BIDS schema can be found in the [bidsschematools][bst desc] documentation,
+and the schema itself may be found at
+<https://bids-specification.readthedocs.io/en/latest/schema.json>.
+(Replace `latest` with a particular version, for example,
+`v1.10.0`, to find a snapshot of the schema at the time of that release.)
+
+## The Schema Validator
+
+The schema validator is now feature complete, validating file names,
+JSON file fields and values, TSV file column names and values,
+and various [checks] that have been written in the schema expression language.
+
+To use the schema validator, visit <https://bids-standard.github.io/bids-validator/>
+or download the [Deno] Javascript runtime and run:
+
+```console
+deno run -A jsr:@bids/validator [PATH]
+```
+
+While many features of the original BIDS validator have been preserved,
+the structure of JSON configuration files and JSON outputs
+(triggered by the `--json` flag) have changed their formats.
+Advanced users may need to update their tooling on either end.
+
+### New features
+
+The schema validator includes features that were never implemented in the
+legacy validator:
+
+-   BIDS Derivatives (introduced in BIDS 1.4.0), including opt-in recursive
+    validation of BIDS Derivatives datasets discovered in `derivatives/`
+    subdirectories.
+
+-   Support for BIDS-MRS (Magnetic Resonance Spectroscopy), introduced in
+    BIDS 1.10.0.
+
+### Behavior changes
+
+-   The inheritance principle is consistently applied for all files.
+
+-   Every JSON field or TSV column defined in BIDS is type-checked against the
+    same source that produces the tables in the specification.
```

Comment:
I am not 100% sure what you mean. So I suspect I won't be the other one.
Can you give an example to disambiguate?